pom.xml

```diff
@@ -13,7 +13,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         
-        <gephi.graphstore.version>0.6.15-viz-engine-SNAPSHOT</gephi.graphstore.version>
+        <gephi.graphstore.version>0.6.15-SNAPSHOT</gephi.graphstore.version>
```

Comment:
`viz-engine` branch of graphstore is still needed to enable spatial index. See https://github.com/gephi/graphstore/compare/master...viz-engine

Reply:
Ah ok it explains +1